src/main/java/com/minecolonies/coremod/blocks/BlockConstructionTape.java

```diff
@@ -254,9 +414,13 @@ public BlockRenderLayer getBlockLayer()
         return BlockRenderLayer.SOLID;
     }
 
+    /**
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
     @SuppressWarnings(DEPRECATION)
     @NotNull
     @Override
+    @Deprecated
     public IBlockState getStateForPlacement(
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1133)